packages/hardhat-core/src/builtin-tasks/test.ts

```diff
@@ -57,6 +57,10 @@ subtask(TASK_TEST_RUN_MOCHA_TESTS)
       mocha.run(resolve);
     });
 
+    // [@types/mocha@7.0.2](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3dce8b667f77c414b123be249d1ea6e98804ea4f/types/mocha/index.d.ts)
+    // is missing the definition of `dispose`, which is actually implemented
+    // https://github.com/mochajs/mocha/blob/v7.2.0/lib/mocha.js#L575
+    // @ts-ignore
```

Comment:
Can we change this to `@ts-expected-error` so that the compiler will shout at us when it is safe to remove the comment?

Reply:
Good point, thanks! I've replaced `@ts-ignore` with `@ts-expect-error`.